docs-ref-services/devtest-labs.md

```diff
@@ -13,22 +13,42 @@ ms.devlang: nodejs
 ms.service: DevTest Labs
 ---
 
-# Azure DevTest Labs modules for Node.js
+# Azure DevTest Labs Package for Node.js
 
 ## Overview
 Azure DevTest Labs is a service that helps developers and testers quickly create environments in Azure while minimizing waste and controlling cost. You can test the latest version of your application by quickly provisioning Windows and Linux environments using reusable templates and artifacts. Easily integrate your deployment pipeline with DevTest Labs to provision on-demand environments. Scale up your load testing by provisioning multiple test agents, and create pre-provisioned environments for training and demos.
 
-## Install the modules with npm
+## Management Package
 
-Use npm to install the Azure DevTest Labs modules for Node.js
-
-### Management
+### Install npm module
 ```bash
 npm install azure-arm-devtestlabs
 ```
 
-##Example
+### Example
+
+This example gets and prints the details of a lab.
+
+```javascript
+const msRestAzure = require('ms-rest-azure');
+const DevTestLabsClient = require('azure-arm-devtestlabs');
+
+const subscriptionId = 'your-subscription-id';
+const resourceGroupName = 'your-resource-group-name';
+const labName = 'your-lab-name';
+
+msRestAzure.interactiveLogin((err, credentials) => {
+  const client = new DevTestLabsClient(credentials, subscriptionId);

```

Comment:
We should be using promises here similar to https://github.com/MicrosoftDocs/azure-docs-sdk-node/pull/91/commits/8c76d0e358b3e4de15b2197cc8ccdd3933baecf3